project_task_category/migrations/10.0.1.0.0/pre-migration.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# © 2018 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+_model_renames = [('project.category.main', 'project.category'), ]
+
+
+@openupgrade.migrate(use_env=True)
+def migrate(env, version):
+    cr = env.cr
+    try:
```

Comment:
Why should fail this?
we might prefer to check if table, column, etc.. exist
My point remove try..except unless we can't control what could happends